core/src/Dscp/Core/Arbitrary.hs

```diff
@@ -118,8 +120,25 @@ instance Arbitrary EducationForm where
     arbitrary = arbitraryBoundedEnum
 
 instance Arbitrary CertificateMeta where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
+    arbitrary = do
+        cmStudentName <- arbitrary
+        cmStudentBirthDate <- arbitrary
+        -- Start year should be after student birth date
+        let getYear date = fromInteger $ toGregorian date ^. _1
+            studentBirthYear = getYear cmStudentBirthDate
+        cmStartYear <- arbitrary `suchThat` (> studentBirthYear)
+        -- End year should be after start year
+        cmEndYear <- arbitrary `suchThat` (> cmStartYear)
+        cmEducationForm <- arbitrary
+        -- Document number should be large enough to resemble
+        -- actual diploma numbers
+        cmNumber <- arbitrary `suchThat` (> 1000000)
+        -- Document should be issued in the year when studies are ended
+        cmIssueDate <- arbitrary `suchThat` (\issueDate -> getYear issueDate == cmEndYear)
```

Comment:
Kek, requests to the endpoint hang. And something in this line makes me guess a suspect.

Reply:
@Martoon-00 resolved. But the removal of `DefPageSize` required some fixes in `servant-util`, check out: https://github.com/serokell/servant-util/pull/3